backend/app/store/comment.go

```diff
@@ -109,7 +109,8 @@ func (c *Comment) SetDeleted(mode DeleteMode) {
 // Sanitize clean dangerous html/js from the comment
 func (c *Comment) Sanitize() {
 	p := bluemonday.UGCPolicy()
-	p.AllowAttrs("class").Matching(regexp.MustCompile("^language-[a-zA-Z0-9]+$")).OnElements("code")
+	p.AllowAttrs("class").Matching(regexp.MustCompile("^chroma$")).OnElements("pre")
+	p.AllowAttrs("class").Matching(regexp.MustCompile(".")).OnElements("span")
```

Comment:
This rule is far too permissive, the proper rule should be "allow `class` property with a particular set of values in `span` inside `pre` tag". I haven't figured out the complete list of possible values so far: CSS file might be a good reference for a full list of possible classes.

Reply:
Oh, and the important part: it seems to me that it's impossible to build such a rule using `bluemonday`.